components/ActivityIndicator/src/MDCActivityIndicator.m

```diff
@@ -206,6 +206,7 @@ - (CGSize)intrinsicContentSize {
 
 - (void)startAnimating {
   if (_animatingOut) {
+    [_delegate activityIndicatorAnimationDidFinish:self];
```

Comment:
You need to check if the delegate has this method—it's optional.

Reply:
Thanks for the catch!